describealign.py

```diff
@@ -1233,8 +1235,15 @@ def error(self, message):
   parser.add_argument("--extension", default="copy",
                       help='File type of output video (e.g. mkv). When set to "copy", copies the ' + \
                            'file type of the corresponding input video. Default is "copy".')
+  parser.add_argument("--install-ffmpeg", action="store_true",
+                      help="Install the required ffmpeg binaries and then exit. This is meant to be" + \
+                           "run from a privileged installer process (e.g. OS X Installer)")
   args = parser.parse_args()
   
+  if args.install_ffmpeg:
+    static_ffmpeg.run.get_or_fetch_platform_executables_else_raise()
```

Comment:
I don't remember exactly what it was, but there was some issues with the Filelock dependency this version of the command uses.  That's why it's the non-locking version elsewhere

Reply:
This was already changed in a later commit to use the existing get_ffmpeg and get_ffprobe functions in describealign